internal/test/integration/rpm_test.go

```diff
@@ -26,6 +26,23 @@ type RpmSuite struct {
 	secondVersionHref string
 }
 
+const testRepoNameWithErrata = "rpm-packages-updateinfo"
+const testRepoWithErrata = "https://fixtures.pulpproject.org/rpm-packages-updateinfo/"
+
+func (r *RpmSuite) CreateTestRepositoryWithErrata(t *testing.T) {
+	_, err := r.client.LookupOrCreateDomain(r.domainName)
+	require.NoError(t, err)
+
+	repoHref, remoteHref, err := r.client.CreateRepository(r.domainName, testRepoNameWithErrata, testRepoWithErrata)
+	require.NoError(t, err)
+
+	syncTask, err := r.client.SyncRpmRepository(repoHref, remoteHref)
+	require.NoError(t, err)
+
+	_, err = r.client.PollTask(syncTask)
+	require.NoError(t, err)
+}
```

Comment:
Could you refactor this to update the existing "CreateTestRepository" method to take a name and URL as input? Then we can use that method for any repository creation.